repo/test/temp_repo.go

```diff
@@ -51,8 +51,14 @@ func newTempRepo(peername, prefix string, g gen.CryptoGenerator) (r TempRepo, er
 		return r, err
 	}
 
+	// Create directory for new Qri repo.
+	QriPath := filepath.Join(RootPath, "qri")
```

Comment:
lower-case this var

Hmmm, why is `RootPath` upper-case?

Reply:
uhhh... dunno. will fix